projects/validation-messages/src/lib/resources/const/validators-with-value.const.ts

```diff
@@ -0,0 +1,7 @@
+import { ValidatorValue, ValidatorWithValue } from '../enums';
+
+export const validatorsWithValue = {
```

Comment:
naming should include information that this is a default Angular Validator mapping

eg, angularValidatorsWithValueMap ?